### PR TITLE
Add HairlineConstraint; apply to InfoCollectionViewCell dividers.

### PR DESCRIPTION
**Pull request by hettiger**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [x] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [x] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) (if applicable)
* [x] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [x] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)

## Description

The hairline constraint is used on top and bottom divider view height constraints to render the separators with 1px height instead of 1pt. This makes the collection view cells look more similar to the grouped table view cells that are being used in many other views of the app.

### CODE REVIEW

**Comment by tkowark** [comment]
Dear @hettiger , as the team is working with almost full capacity on implementing all the backlog items, please excuse some delays in the PR review. We hope that there might be more time for that after the weekend.

Would be great if you could keep it up to date with changes to the base branch in the meantime. For further PRs, we kindly ask to only provide ones for issues that the team marked for community contributions and assigned to you. Thanks for your understanding.

File: src/xcode/ENA/ENA/Source/Views/HairlineConstraint.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+// under the License.
+//
+
+import UIKit
+
+class HairlineConstraint: NSLayoutConstraint {
+	
+	override func awakeFromNib() {
+		super.awakeFromNib()
+		self.constant = 1.0 / UIScreen.main.scale
```
Comment (ChristianKienle) [inline src/xcode/ENA/ENA/Source/Views/HairlineConstraint.swift:26]: What do you think about this:

```suggestion
		constant = 1.0 / ((firstItem as? UIView)?.window?.screen.scale) ?? UIScreen.main.scale
```

**Comment by EisvogelSAP** [comment]
Hey @tkowark, thanks for your contribution. All hairlines/separators in the app will be aligned in due course.

File: src/xcode/ENA/ENA/Source/Views/HairlineConstraint.swift (hunk 1 above)
Comment (hettiger) [reply src/xcode/ENA/ENA/Source/Views/HairlineConstraint.swift:26]: Thanks for the review @ChristianKienle 

Your code does not compile but I'll apply a similar change.

**Comment by EisvogelSAP** [comment]
Will be resolved by #266.

**Pull request closed without merging**